controllers/candidatesController.js

```diff
@@ -1,93 +1,86 @@
 const Candidate = require("../models/candidate");
-const User = require("../models/user");
-const app = require("express")();
 
 module.exports = {
-  index: (req, res, next) => {
-    Candidate.find({})
-      .exec()
-      .then((candidates) => {
-        res.locals.candidates = candidates;
-        next();
-      })
-      .catch((error) => {
-        console.log(`Error fetching candidates: ${error.message}`);
-        return [];
-      })
-      .then(() => {
-        console.log("promise complete");
-      });
-  },
+    index: (req, res, next) => {
+        Candidate.find({})
+            .exec()
+            .then((candidates) => {
+                res.locals.candidates = candidates;
+                next();
+            })
+            .catch((error) => {
+                console.log(`Error fetching candidates: ${error.message}`);
+                return [];
+            })
+            .then(() => {
+                console.log("promise complete");
+            });
+    },
 
-  indexView: (req, res) => {
-    res.render("candidates/index");
-  },
+    indexView: (req, res) => {
+        res.render("candidates/index");
+    },
 
-  new: (req, res) => {
-    res.render("candidates/new");
-  },
+    new: (req, res) => {
+        res.render("candidates/new");
+    },
 
-  create: (req, res, next) => {
-    let candidateParams = {
-      preferred_position: req.body.preferred_position,
-      soft_skills: req.body.soft_skills,
-      other_aspects: req.body.other_aspects,
-      work_culture_preferences: req.body.work_culture_preferences,
-    }
-    Candidate.create(candidateParams)
-      .then(candidate => {
-        req.flash('success', `${candidate.preferred_position} candidate created successfully!`);
-        res.locals.redirect = '/candidates';
+    //CREATE
+    create: (req, res, next) => {
+		let candidateParams = {
+            preferred_position: req.body.preferred_position,
+            soft_skills: req.body.soft_skills,
+            other_aspects: req.body.other_aspects,
+            work_culture_preferences: req.body.work_culture_preferences,
+        }
+		Candidate.create(candidateParams)
+			.then(candidate => {
+				res.locals.redirect = '/candidates';
         res.locals.candidate = candidate;
-        next();
-      })
-      .catch(error => {
-        console.log(`Error saving candidate profile: ${error.message}`);
-        res.locals.redirect = "/candidates/new";
-        req.flash(
-          "error",
-          `Failed to create user account because: ${error.message}.`
-        );
-        next();
+        console.log('a candidate saved into both MongoDB and Elasticsearch')
```

Comment:
Once again, why did you delete the flash message & in this case change the structure?
Also, why was the redirect deleted?